### ISSUE TITLE
[Feature Request] Conversion between Big and BigInt

### ISSUE THREAD

**Issue by michaelhkay** [MikeMcl/big.js#186]
It would be great to be able to convert between Big and BigInt (in both directions) without going via a string representation.

**Comment by MikeMcl** [comment]
Yes, it would, but it would need to be done via string representation internally anyway, and I would prefer not to add any non-ES3 features.

**Comment by nycdotnet** [comment]
`typeof 5n === “bigint”` should work in an ES3 environment but allow handling bigints in environments that supported it.
